crates/matrix-sdk/src/sync.rs

```diff
@@ -156,6 +156,12 @@ impl Client {
         Ok(response)
     }
 
+    /// Calls event handlers and notification handlers after a sync response has
+    /// been processed.
+    ///
+    /// At this point, the sync response's data has been taken into account and
+    /// persisted in the store, if needs be. This function is only calling
+    /// the event, room update and notification handlers.
```

Comment:
Thanks for this. I wonder if the method should get a new name that's more clearly separated from the sync processing method(s) in `matrix-sdk-base`.

Reply:
Yes, I thought about that too. It's hard to find a good name, maybe `call_handlers` or something longer like `call_handlers_for_sync_response`; I don't find these very satisfying.